server/utils/courseParticipationUtils.test.ts

```diff
@@ -211,28 +209,70 @@ describe('CourseParticipationUtils', () => {
+          describe('and there is a yearStarted value', () => {
+            const withOutcomeYearStarted = {
+              ...courseParticipationWithName,
+              outcome: { status: 'incomplete' as CourseParticipationOutcome['status'], yearStarted: 2019 },
+            }
 
-        const { rows } = CourseParticipationUtils.summaryListOptions(withoutOutcomeYearStarted, referralId)
-        const outcomeRow = getRow(rows, 'Outcome')
+            it('displays the outcome and yearStarted value', () => {
+              const { rows } = CourseParticipationUtils.summaryListOptions(withOutcomeYearStarted, referralId)
+              expect(getRowValueText(rows, 'Outcome')).toEqual('Incomplete, year started 2019')
+            })
+          })
 
-        expect(outcomeRow).toEqual({ key: { text: 'Outcome' }, value: { text: 'Incomplete' } })
-      })
+          describe('and there is no yearStarted value', () => {
+            const withoutOutcomeYearStarted = {
+              ...courseParticipationWithName,
+              outcome: { status: 'incomplete' as CourseParticipationOutcome['status'], yearStarted: undefined },
+            }
+
+            it('displays the outcome on its own', () => {
+              const { rows } = CourseParticipationUtils.summaryListOptions(withoutOutcomeYearStarted, referralId)
+              expect(getRowValueText(rows, 'Outcome')).toEqual('Incomplete')
+            })
+          })
+        })
+
+        describe('when the outcome is complete', () => {
+          describe('and there is a yearCompleted value', () => {
+            const withOutcomeYearCompleted = {
+              ...courseParticipationWithName,
+              outcome: { status: 'complete' as CourseParticipationOutcome['status'], yearCompleted: 2019 },
+            }
+
+            it('displays the outcome and yearCompleted value', () => {
+              const { rows } = CourseParticipationUtils.summaryListOptions(withOutcomeYearCompleted, referralId)
+              expect(getRowValueText(rows, 'Outcome')).toEqual('Complete, year completed 2019')
+            })
+          })
 
-      it('only shows the status in the Outcome row when yearCompleted is undefined on a complete outcome', () => {
-        const withoutOutcomeYearCompleted = {
-          ...courseParticipationWithName,
-          outcome: { status: 'complete' as CourseParticipationOutcome['status'], yearCompleted: undefined },
-        }
+          describe('and there is no yearCompleted value', () => {
+            const withoutOutcomeYearCompleted = {
+              ...courseParticipationWithName,
+              outcome: { status: 'complete' as CourseParticipationOutcome['status'], yearCompleted: undefined },
+            }
 
-        const { rows } = CourseParticipationUtils.summaryListOptions(withoutOutcomeYearCompleted, referralId)
-        const outcomeRow = getRow(rows, 'Outcome')
+            it('displays the outcome and yearStarted value', () => {
```

Comment:
`'displays the outcome status on its own'` (or just `status` depending on previous comment)